### PR TITLE
Adapt hand-written to RC2

### PR DESCRIPTION
**Pull request by adriaanm**
submitted to scala-lang https://github.com/scala/scala-lang/pull/494

### CODE REVIEW

File: hand-written.md [hunk 1]
```diff
@@ -20,35 +33,22 @@ The most notable changes since M5 are:
...
-  `scala.tools.nsc.backend.jvm.BCodeSkelBuilder`; see
-  [#5395](https://github.com/scala/scala/pull/5395).
-
-Because of the last-mentioned regression,
-[Shapeless](https://github.com/milessabin/shapeless) will not be
-published for 2.12.0-RC1.
+trait-based code run slower. We've investigated this issue in depth,
+and have implemented important improvements in RC2.
+Compile times may still be longer in 2.12 than 2.11. Please let us know
+if you notice any perfomance regresions. We will continue to tweak
```
Comment (dwijnand) [inline hand-written.md]: performance and regressions

Comment (adriaanm) [reply hand-written.md]: the linebreak in the markdown is 
unfortunate, but it does
continue :-)

Comment (dwijnand) [reply hand-written.md]: Uhh.. I was just pointing out two typos..

s/perfomance/performance/
s/regresions/regressions/

Comment (adriaanm) [reply hand-written.md]: Aha! thanks.

**Pull request merged**